### PR TITLE
Added method to retrieve all tags used in asset repository

### PR DESCRIPTION
**Pull request by blakecodes**

### CODE REVIEW

File: csharp/Squidex.ClientLibrary/Squidex.ClientLibrary/SquidexAssetClient.cs [hunk 1]
```diff
@@ -127,6 +130,24 @@ public async Task<Asset> UpdateAssetContentAsync(string id, string contentName,
...
+        {
+            var jsonData = JsonConvert.SerializeObject(data);
+
+            var request = new StringContent(jsonData, Encoding.UTF8, "application/json");
+            var response = await RequestAsync(HttpMethod.Put, BuildAppAssetsUrl(id), request);
+
+            return response.StatusCode;
+        }
+
+        public async Task<Dictionary<string, string>> GetAssetsTagsAsync()
```
Comment (SebastianStehle) [inline csharp/Squidex.ClientLibrary/Squidex.ClientLibrary/SquidexAssetClient.cs]: Dictionary<string, int> would be better.

Comment (blakecodes) [reply csharp/Squidex.ClientLibrary/Squidex.ClientLibrary/SquidexAssetClient.cs]: Fixed.

**Comment by blakecodes** [comment]
@SebastianStehle added another commit to handle Updating with the AsDraft flag.

**Comment by SebastianStehle** [comment]
I also published it to Nuget

**Pull request merged**